### PR TITLE
Support geo_shape visualization in documents layer

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
Support geo_shape visualization on documents layer in GeoJSON type.

### Issues Resolved
https://github.com/opensearch-project/dashboards-maps/issues/100

### Demo
https://user-images.githubusercontent.com/90288540/206264030-86b0fdba-e7a7-498f-bae8-fc787b4aa1ce.mov

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: maps_dashboards/public/components/layer_config/documents_config/document_layer_style.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      setHasInvalidThickness(true);
+    } else {
+      setHasInvalidThickness(false);
+    }
+  }, [borderThickness]);
+
+  const toggleButtonsGeoType = [
+    {
+      id: `${geoTypeToggleButtonGroupPrefix}__Point`,
+      label: 'Point',
```
Comment (VijayanB) [inline maps_dashboards/public/components/layer_config/documents_config/document_layer_style.tsx:85]:
```suggestion
      label: 'Points',
```

File: maps_dashboards/public/components/layer_config/documents_config/document_layer_style.tsx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    },
+    {
+      id: `${geoTypeToggleButtonGroupPrefix}__Line`,
+      label: 'Line',
+    },
+    {
+      id: `${geoTypeToggleButtonGroupPrefix}__Polygon`,
+      label: 'Polygon',
```
Comment (VijayanB) [inline maps_dashboards/public/components/layer_config/documents_config/document_layer_style.tsx:93]:
```suggestion
      label: 'Polygons',
```

**Comment by junqiu-lei** [comment]
> Shall we add Marker Size to points ?

Agree, on UX mock, I didn't find this option, will update PR to add.

**Comment by junqiu-lei** [comment]
By default, marker size is 5.
image

**Review by VijayanB — APPROVED** [review]

**Review by martin-gaievski — APPROVED** [review]
looks good, thx.

**Review by VijayanB — APPROVED** [review]

**Review by martin-gaievski — APPROVED** [review]

**Pull request merged**